meson_options.txt

```diff
@@ -86,212 +86,212 @@ option('cfi', type: 'boolean', value: 'false',
        description: 'Control-Flow Integrity (CFI)')
 option('cfi_debug', type: 'boolean', value: 'false',
        description: 'Verbose errors in case of CFI violation')
-option('multiprocess', type: 'feature', value: 'auto',
+option('multiprocess', type: 'feature', value: 'disabled',
        description: 'Out of process device emulation support')
 option('vfio_user_server', type: 'feature', value: 'disabled',
        description: 'vfio-user server support')
-option('dbus_display', type: 'feature', value: 'auto',
+option('dbus_display', type: 'feature', value: 'disabled',
        description: '-display dbus support')
-option('tpm', type : 'feature', value : 'auto',
+option('tpm', type : 'feature', value : 'disabled',
        description: 'TPM support')
 
 # Do not enable it by default even for Mingw32, because it doesn't
 # work on Wine.
 option('membarrier', type: 'feature', value: 'disabled',
        description: 'membarrier system call (for Linux 4.14+ or Windows')
 
-option('avx2', type: 'feature', value: 'auto',
+option('avx2', type: 'feature', value: 'disabled',
```

Comment:
why avx2 disabled?

Reply:
I wasn't so sure about that one. It should be installed by default I think, but I couldn't confirm that it actually is.

Reply:
I reactivated it as it seems to not be depended on a library (https://github.com/AFLplusplus/qemu-libafl-bridge/blob/main/meson.build#L2237-L2263) which we would need to link in libafl